pom.xml

```diff
@@ -19,24 +19,67 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.eclipse.jnosql</groupId>
-        <artifactId>jnosql-parent</artifactId>
-        <version>1.0.0-b6-SNAPSHOT</version>
+        <groupId>org.eclipse.ee4j</groupId>
```

Comment:
@dearrudam, in this case, it is acceptable to use the JNoSQL parent once both belong to the same project: Eclipse JNoSQL.
So, please keep as jnosql

Reply:
@otaviojava oh, you're right! I could change the parent direct to:
```xml
   <parent>
        <groupId>org.eclipse.jnosql.mapping</groupId>
        <artifactId>jnosql-mapping-parent</artifactId>
        <version>1.0.0-SNAPSHOT</version>
    </parent>
```
Am I right?

Reply:
@otaviojava, thanks for notifying me! I'm fixing it!